app/mentor/models.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from economy.models import SuperModel
+from rest_framework import serializers
+
+
+class Mentor(SuperModel):
+
+    TIME_RANGE = [
+        ('1_5', '1 - 5'),
+        ('5_10', '5 - 10'),
+        ('10_15', '10 - 15'),
+        ('15_20', '15 - 20'),
+        ('20_', '20+'),
+    ]
+
+    profile = models.ForeignKey('dashboard.Profile', related_name='mentor', on_delete=models.CASCADE)
+    name = models.CharField(max_length=255, blank=True)
+    email = models.CharField(max_length=255)
```

Comment:
We can probably get away with excluding `name` and `email` from `Mentor` since we have `profile.user`

Reply:
ok, then these fields will be removed from form or they will become readonly i suppose